### ISSUE TITLE
postfixadmin_admin does not seem to work properly

### ISSUE THREAD

**Issue by e100** [onddo/postfixadmin-cookbook#6]
If the admin being created does not exist and encrypted attributes are enabled an exception is thrown.

In the admin LWRP the definition of password seems incorrect.

```ruby
# providers/admin.rb
def password
  if encrypted_attributes_enabled?
    Chef::EncryptedAttribute.load(new_resource.password)
  else
    new_resource.password
  end
end
```

The resource only allows a string yet the code wants different data depending on if encrypted attributes are enabled or not.

I think the check for encrypted attributes is not needed here, it should simply be:

```ruby
def password
  new_resource.password
end
```

**Comment by zuazo** [comment]
@e100 thanks for reporting. Fixed in master (7ca12ba058c46c55d76e94eac00d28353139e8ba).

**Comment by zuazo** [comment]
Fix released in [`2.1.0`](https://github.com/zuazo/postfixadmin-cookbook/releases/tag/2.1.0).
